src/components/signoff/SimpleReview/PerRecordDiffView.tsx

```diff
@@ -160,6 +166,34 @@ function DiffLabel({ changeType }: { changeType: ChangeType }) {
   }
 }
 
+export function formatDiffHeader({
+  source,
+  target,
+  displayFields = [],
+}: {
+  source: ValidRecord;
+  target: ValidRecord;
+  displayFields: string[];
+}) {
+  let fields = [];
+
+  for (let f of displayFields) {
+    fields.push(
+      <span>
+        <label>{f}:</label> {renderDisplayField(target || source, f)}
```

Comment:
I have a doubt about this. `target` would be `undefined` when record is deleted?  Would it be a tombstone? (eg. `{id: "xxx", deleted: true}`)

Reply:
If the record was deleted, target would be undefined (or at the very least nullish) so we would render what the source object had for the property.

Reply:
Example screenshot
![image](https://github.com/Kinto/kinto-admin/assets/148472676/e258a4af-1609-4432-8d35-3bae780e61ac)